### ISSUE TITLE
[1.38.0-beta.1] Blue fails contrast on light gray

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1467]
I'm not sure how the new colors in 7.0 will impact this, at all, but the current blue fails contrast on the light gray background:

<img width="1519" height="731" alt="Image" src="https://github.com/user-attachments/assets/c8c8aed1-57fc-49ed-a1ef-b3e37f5c5be5" />

It really only just barely passes on white or the light blue in the sidebar active tab background.

This blue should be made significantly darker, maybe even trying for AAA.

It looks like it may be coming from core, so perhaps it needs to be retested with 7.0 to see if it passes then? Or we just override with our own colors.
